packages/custom-functions-metadata/test/cases/field/expected.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
How is this testing the new fields?

Reply:
> How is this testing the new fields?

When add a @field in functions.js or functions.ts file with the fieldname as the first comment and value as the second comment, it will generate JSON file with the metadata named fieldname and valued the value.